sherpa/ui/utils.py

```diff
@@ -12105,39 +12161,85 @@ def get_kernel_plot(self, id=None, recalc=True):
     # Line plots
     #
 
-    def _multi_plot(self, args, plotmeth='plot', **kwargs):
+    def _multi_plot(self, args, plotmeth="plot", **kwargs):
+        """Handle the plot() or contour() call.
+
+        The arguments are split up into groups - a "command" followed
+        by optional arguments - and then each group is used to create
+        a plot/contour.
+
+        Parameters
+        ----------
+        args : list
+            The arguments to the call. It can not be empty.
+        plotmeth : {"plot", "contour"}
+            The call.
+        kwargs
+            The keyword arguments to apply to each plot or contour.
+
+        Notes
+        -----
+        Each group consists of the name of the plot/contour, followed
+        by optional arguments. So
+
+            self._multi_plot(["data", "data", "org"], plotmeth="plot")
+
+        has two "data" style plots, the first with the default
+        identifier and the second with the identifer "org". It is the
+        need to scan through the arguments that leads to the ban on
+        identifiers matching the plot/contour "command names".
+
+        Each plot/contour command matches a key in the
+        _plot/contour_type_names dictionary, and the value is used to
+        get the underlying "plot objet" via get_<value>_plot or
+        get_<value>_contour.
+
+        Multiple arguments can be sent, but they rely on positional
+        ordering only, so
+
+            args = ["model_component", mdl, "model_component", 2, mdl]
+            plotmeth = "plot"
+
+        only works if
+
+            get_model_component_plot(mdl)
+            get_model_component_plot(2, mdl)
+
+        are meaningful calls.
+
+        """
+
+        # This is an internal routine so it is not expected to be
+        # called incorrectly, but make sure we catch any such problem.
+        #
```

Comment:
This comment is added to mainly point out why the two `raise ArgumentxxxErr` calls are not covered by tests.